### PR TITLE
Cargo Console Spellcheck

### PR DESCRIPTION
**Pull request by BonniePandora**
## About The Pull Request

Spellchecks all the descriptions and names of the crates that can be ordered from cargo.
Two commits to it cos I made a slight whoopsy with the cutlery crate on my first pass.

## Why It's Good For The Game

Quality of life improvements, always nice to not see typos in a supposedly cutting edge space station.

## Changelog
:cl:
fix: Typos in multiple .dm's from the /modules/cargo/packs/ folder have been sorted
/:cl:

### CODE REVIEW

**Comment by BonniePandora** [comment]
In retrospect these are prolly more tweaks than outright fixes but uh, too late to change that now

File: code/modules/cargo/packs/misc.dm [hunk 1]
```diff
@@ -162,7 +162,7 @@
 
 /datum/supply_pack/misc/lewd
 	name = "Lewd Crate" // OwO
-	desc = "Psss want to have a good time with your sluts? Well I got what you want maid clothing, dildos, collars and more!"
+	desc = "Pssst, want to have a good time with your sluts? Well I got what you want; maid clothing, dildos, collars and more!"
```
Comment (Putnam3145) [inline code/modules/cargo/packs/misc.dm]: probably should be a colon instead of a semicolon (sorry)

File: code/modules/cargo/packs/organic.dm [hunk 2]
```diff
@@ -335,7 +335,7 @@
 
 /datum/supply_pack/organic/vday
 	name = "Surplus Valentine Crate"
-	desc = "Turns out we got warehouses of this love-y dove-y crap. Were sending out small barged buddle of Valentine gear. This crate has two boxes of chocolate, three poppy flowers, five candy hearts, and three cards."
+	desc = "Turns out we got warehouses of this love-y dove-y crap. We're sending out small bargin buddles of Valentine gear. This crate has two boxes of chocolate, three poppy flowers, five candy hearts, and three cards."
```
Comment (Putnam3145) [inline code/modules/cargo/packs/organic.dm]: bargain bundles, not bargin buddles (probably)

File: code/modules/cargo/packs/service.dm [hunk 3]
```diff
@@ -123,7 +123,7 @@
 
 /datum/supply_pack/service/janitor/janpremium
 	name = "Janitor Premium Supplies"
-	desc = "Do to the union for better supplies, we have desided to make a deal for you, In this crate you can get a brand new chem, Drying Angent this stuff is the work of slimes or magic! This crate also contains a rag to test out the Drying Angent magic, three wet floor signs, and some spare bottles of ammonia."
+	desc = "Due to the Janitorial Union for Greater Supplies, we have decided to make a deal for you, In this crate you can get a brand new chem, Drying Angent! This stuff is the work of slimes or magic! This crate also contains a rag to test out the Drying Angent magic, three wet floor signs, and some spare bottles of ammonia."
```
Comment (Putnam3145) [inline code/modules/cargo/packs/service.dm]: drying angent

File: code/modules/mob/living/emote.dm [hunk 4]
```diff
@@ -151,6 +151,12 @@
 	emote_type = EMOTE_AUDIBLE
 	stat_allowed = UNCONSCIOUS
 
+/datum/emote/living/gekker//Planning on adding audio and a race check for felinids at a later date but for now this'll do
```
Comment (Putnam3145) [inline code/modules/mob/living/emote.dm]: i'm pretty sure this is spelled "gecker", and besides that it can probably go in its own PR

**Review by Putnam3145 — CHANGES_REQUESTED** [review]
and yeah i know this is all pedantic but this is a pedantic PR, so I figure further pedantry is warranted

File: code/modules/cargo/packs/misc.dm (hunk 1 above)
Comment (BonniePandora) [reply code/modules/cargo/packs/misc.dm]: Was going off the basis that semicolons are used for at the beginning of lists of things in sentences, but fair enough

File: code/modules/mob/living/emote.dm (hunk 4 above)
Comment (BonniePandora) [reply code/modules/mob/living/emote.dm]: If I'd been meaning the noises baby monkeys make, aye it'd be spelt that way. 
But it's meant to be the noises foxes make, if you check Renaults emotes you'll see that he has a chance of gekkering too, spelt much the same way.
I'll get shot of this and make it it's own PR instead.

**Comment by Owai-Seek** [comment]
Not trying to be a downer, but I did modify a bunch of these same files in #9912 and I think these might conflict when one gets merged.

**Comment by BonniePandora** [comment]
Aight with Owai merging the typos I caught that he missed into his own PR, I'm gonna go ahead and close this.
